src/account/utils/signature.cairo

```diff
@@ -4,10 +4,15 @@
 use ecdsa::check_ecdsa_signature;
 use openzeppelin::account::interface::EthPublicKey;
 use openzeppelin::account::utils::secp256k1::Secp256k1PointPartialEq;
-use starknet::eth_signature::Signature;
-use starknet::secp256_trait::{is_signature_entry_valid, recover_public_key};
+use starknet::secp256_trait;
 use starknet::secp256k1::Secp256k1Point;
```

Comment:
We don't need `Secp256k1PointPartialEq` or `Secp256k1Point` in here anymore, right?